TypeCobol/Compiler/Symbols/IndexSymbol.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.IO;
+
+namespace TypeCobol.Compiler.Symbols
+{
+    /// <summary>
+    /// A Symbol that represents an Index.
+    /// The type of an index is : S9(8) comp
+    /// </summary>
+    public class IndexSymbol : VariableSymbol
+    {
+        /// <summary>
+        /// Named constructor
+        /// </summary>
+        /// <param name="name">The Index name</param>
+        public IndexSymbol(string name)
+            : base(name, Kinds.Index)
+        {
+            base.Type = Builtins.IndexType;
+        }
+
+        /// <summary>
+        /// The Indexed Variable.
+        /// </summary>
+        public VariableSymbol Indexed { get; set; }        
```

Comment:
We now Indexed == Owner, so I think we can remove this property, or at least make it returns Owner.

Reply:
Turned it into a read-only property returning the Owner (a06915bae5364c522b05750871eb53939fa7734e).